src/main/java/codeu/model/store/basic/UserStore.java

```diff
@@ -66,7 +74,7 @@ private UserStore(PersistentStorageAgent persistentStorageAgent) {
     users = new ArrayList<>();
 
     // hard-coded initial Admin:
-    this.addUser("Admin01", "AdminPass01", /*admin=*/ true);
```

Comment:
The comment was left there so the code is easier to understand; without the comment, it's hard to tell what boolean indicates as there is no further information

Reply:
Oh I did not mean to remove that, sorry. Dan told me this last time. I think I made a mistake when I was merging branches. I will put it back.